avocado/plugins/spawners/podman.py

```diff
@@ -89,18 +93,22 @@ def is_task_alive(self, runtime_task):
     async def deploy_artifacts(self):
         pass
 
-    def deploy_avocado(self):
-        pass
-
-    async def spawn_task(self, runtime_task):
-
-        podman_bin = self.config.get('spawner.podman.bin')
-        if not os.path.exists(podman_bin):
-            msg = 'Podman binary "%s" is not available on the system'
-            msg %= podman_bin
-            runtime_task.status = msg
+    async def deploy_avocado(self, where):
```

Comment:
This specific line should be part of the previous commit that changes the method signature.